cypress/integration/my-orcid-sidebar.e2e.js

```diff
@@ -0,0 +1,118 @@
+describe('My Orcid sidebar' + runInfo(), () => {
+  describe('Orcid id', () => {
+    it('display the users Orcid', () => {})
+    it('display url to navigate to the public page view', () => {})
+  })
+  describe('Emails' + runInfo(), () => {
+    it('display a user with only a primary email "unverified"', () => {})
+    it('display a user with only a primary email "verified"', () => {})
+    it('display multiple emails "unverified" and "verified"', () => {
+      // Expect changes to be display outside and inside of the modal
+    })
+    describe('Emails edit modals' + runInfo(), () => {
+      it('add additional emails and display those as "unverified" with default privacy', () => {
+        // Expect changes to be display outside and inside of the modal
+      })
+      it('remove emails', () => {
+        // Expect the primary email to not display the delete button
+        // Expect changes to be display outside and inside of the modal
+      })
+      it('edit a non-primary and display it as "unverified"', () => {
+        // Expect changes to be display outside and inside of the modal
+      })
+      it('edit a primary email and display it as "verified"', () => {
+        // Expect changes to be display outside and inside of the modal
+      })
+      it('display multiple emails "unverified" and "verified"', () => {
+        // Expect changes to be display outside and inside of the modal
+      })
+      it('resend the email verification email', () => {})
```

Comment:
Should we show an alert message right after a verification email is resend?

Reply:
As we have concluded, yes an alert message is show right after, as show on the new sketches zpl://screen?pid=5ee88fa6b1d4aa97af37a43a&sid=60085d04156ab30330554f3a